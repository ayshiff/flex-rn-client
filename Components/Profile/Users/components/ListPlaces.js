import { List } from "react-native-elements";
import React from "react";

const ListPlaces = (props: { handleList: any, prop1: any => null }) => {
  const { handleList, prop1 } = props;
  return (
    <List containerStyle={{ marginBottom: 20 }}>{handleList.map(prop1)}</List>
  );
};

export default ListPlaces;
