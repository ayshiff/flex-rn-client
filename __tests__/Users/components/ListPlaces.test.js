import { View } from "react-native";
import React from "react";
import renderer from "react-test-renderer";
import * as enzyme from "enzyme";
import ReactSixteenAdapter from "enzyme-adapter-react-16";
import ListPlaces from "../../../Components/Profile/Users/components/ListPlaces";

jest.useFakeTimers();
enzyme.configure({ adapter: new ReactSixteenAdapter() });

it("renders correctly", () => {
  const wrapper = renderer
    .create(
      <ListPlaces
        key={1}
        handleList={[1, 2, 3, 4, 5]}
        prop1={event => <View>{event}</View>}
      />
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
