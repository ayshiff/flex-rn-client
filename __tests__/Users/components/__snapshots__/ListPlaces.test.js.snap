// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#bbb",
        "borderTopWidth": 1,
        "marginTop": 20,
      },
      Object {
        "marginBottom": 20,
      },
    ]
  }
>
  <View>
    1
  </View>
  <View>
    2
  </View>
  <View>
    3
  </View>
  <View>
    4
  </View>
  <View>
    5
  </View>
</View>
`;
